src/hotspot/share/gc/shenandoah/shenandoahGenerationalHeap.cpp

```diff
@@ -131,18 +131,15 @@ void ShenandoahGenerationalHeap::initialize_heuristics() {
   // Initialize global generation and heuristics even in generational mode.
   ShenandoahHeap::initialize_heuristics();
 
-  // Max capacity is the maximum _allowed_ capacity. That is, the maximum allowed capacity
-  // for old would be total heap - minimum capacity of young. This means the sum of the maximum
+  // Max capacity is the maximum _allowed_ capacity. This means the sum of the maximum
```

Comment:
I don't understand the relevance of this comment.  Is there still a mximum allowed for old and a maximum allowed for young?

Reply:
This comment stemmed from own confusion over fields and variables called _max_ `capacity` . I would like to rename the `_max_capacity` field to just `_capacity`. In my mind, the _max_ should be immutable, but that isn't how Shenandoah uses this field.